src/dataset_view.h

```diff
@@ -105,21 +105,25 @@ template <class Tag> struct SubdataHelper<Bin<Tag>> {
 };
 
 /// Class with overloads used to handle "virtual" variables such as
-/// Coord::SpectrumPosition.
+/// Coord::Position.
 template <class Tag> struct ItemHelper {
   static element_return_type_t<Tag> get(const ref_type_t<Tag> &data,
                                         gsl::index index) {
     return data[index];
   }
 };
 
-template <> struct ItemHelper<Coord::SpectrumPosition> {
-  static element_return_type_t<Coord::SpectrumPosition>
-  get(const ref_type_t<Coord::SpectrumPosition> &data, gsl::index index) {
+// Note: Special case! Coord::Position can be either derived based on detectors,
+// or stored directly.
+template <> struct ItemHelper<const Coord::Position> {
```

Comment:
Would be nice to have a code example maybe in test of this working.

Reply:
See `test/example_instrument_test.cpp`.